package.json

```diff
@@ -62,6 +62,7 @@
     "sw-precache": "^1.2.6",
     "through2": "0.6.3",
     "vinyl-map": "1.0.1",
+    "web-component-tester": "^3.1.3",
```

Comment:
I don't think this is needed unless readme will say to execute `node_modules/web-component-tester/bin/wct` to run the tests or we'll incorporate it in e.g. `gulp test`.